api/src/pcapi/alembic/versions/20230801T090635_a55a582f209d_remove_offerer_columns.py

```diff
@@ -0,0 +1,29 @@
+"""drop offerer unused columns"""
+from alembic import op
+
+from pcapi import settings
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "a55a582f209d"
+down_revision = "e03f70489edb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # Theses commands should be quick however acquiring the lock on the table
+    # offerer can take a long time. The lock will be acquired by the first
+    op.execute("""SET SESSION lock_timeout = '90s'""")
+    op.execute("""ALTER TABLE "offerer" DROP CONSTRAINT IF EXISTS "offerer_idAtProviders_key";""")
+    op.execute("""ALTER TABLE "offerer" DROP COLUMN IF EXISTS "idAtProviders";""")
+    op.execute("""ALTER TABLE "offerer" DROP COLUMN IF EXISTS "lastProviderId";""")
+    op.execute("""ALTER TABLE "offerer" DROP COLUMN IF EXISTS "thumbCount";""")
```

Comment:
`thumbCount` et `fieldsUpdated` sont NOT NULL. Il faudrait rendre ces colonnes NULLable dans une migration pré-déploiement. (Sinon le code pourrait échouer entre le moment où le code est déployé et le moment où cette migration est jouée.)